ttg/ttg/util/region_probe.h

```diff
@@ -0,0 +1,264 @@
+#ifndef TTG_REGION_PROBE_H
+#define TTG_REGION_PROBE_H
+
+#include <string>
+#include <vector>
+#include <cassert>
+
+#include <parsec/profiling.h>
```

Comment:
should this be protected by ifdefs to avoid having to introduce dependence of `ttg` target on parsec targets. Alternatively we can make parsec a hard dependence of TTG, but that may be undesirable long term.

Reply:
I don't think we have a `HAVE_PARSEC` preprocessor flag right now and both the madness and parsec backends provide access to the parsec header/libs. If we create a backend that does not depend on PaRSEC we'll have to revisit that.